railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
+    <meta charset="utf-8" />
```

Comment:
It is intentional that there is no charset. Please see https://github.com/rails/rails/commit/ba0407337e93c4ef55cef3472143f62e8a984a64#commitcomment-4068989